src/components/MenuItem.js

```diff
@@ -134,7 +135,7 @@ const MenuItem = (props) => {
                                     style={titleTextStyle}
                                     numberOfLines={1}
                                 >
-                                    {props.title}
+                                    <TextEmoji isMenuItem style={[styles.emojiMessageText, styles.profileEmojiText]}>{props.title}</TextEmoji>
```

Comment:
```suggestion
                                    <TextEmoji isMenuItem style={[styles.emojiMessageText, styles.profileEmojiText]}>{title}</TextEmoji>
```

if possible may be we can destructure the props , something like this
`const { title } = props` , this way it becomes more clearer in the beginning as , what all props are we going to use at one glance

Reply:
@avijeetpandey from the looks of things destructuring props isn't often found written within the project, as a majority of the components use props directly, pretty much all props are passed and used on the `MenuItem` component directly.
@parasharrajat @roryabraham please advise if I should destructure just the `title ` prop from the props object should we do this the code structure might change on the component and in other to unify this we might have to do the same for every single-props used within the component

Reply:
@josemak25 I completely agree, doing this in one place, won't do good, but following this in the entire codebase will definitely increase code quality and readability.